main.tex

```diff
@@ -104,6 +104,30 @@ \subsection{Übersicht}
 In der Zwischenzeit versuchte Facebook, nach initellen Startschwierigkeiten, die Initiative durch Rebranding zu retten, was aber nicht reichte um die kritische Öffentlichkeit zu beeinflußen, deren Meinung bis zu den Entscheidungträgern der zuständigen Behörde vordrigen konnte.
 Welche sich dann mit einem indirekten Verbot auf der Seite der Protestierenden positionierte.
 
+% 2.5 Nationaler Start (Google leaves, public opinion, TRAI accusations, FB urges user to work for them)
+\subsection{Nationaler Start}
+
+% 2.6 - Verbot durch Regulatoren (TRAIs "Prohibitions of Discriminiatory tariffs for Data Services Regulation 2016", aftermath)
+\subsection{Verbot durch Regulatoren}
+
+%TRAI in den Index?
+
+%Politics of Free Basics and its Death in India p7
+
+%Internet.org suspended durch TRAI in 2015
+%TRAI schuldigt Facebook der Täuschung der
+
+Noch in Mitten des Austausches über das Pro und Contra der Netzneutralität veranlasste TRAI im Dezember 2015 eine Anordnung an Reliance Communications Facebook's Free Basics vorläufig zu sperren.
```

Comment:
```suggestion
Noch in Mitten des Austausches über das Pro und Contra der Netzneutralität veranlasste TRAI (Telecom Regulatory Authority of India) im Dezember 2015 eine Anordnung an Reliance Communications Facebook's Free Basics vorläufig zu sperren.
```

Damit man weiß wofür TRAI steht

Reply:
achja, da war ich mir noch nicht sicher, ob das schon vorher erwähnt wird oder erst hier

Reply:
Müsste aber mindestens schon in 2.5 erwähnt werden